docs/semgrep-code/findings.md

```diff
@@ -62,7 +68,7 @@ Use filters to narrow down your results. The following criteria are available fo
 | **Status**             | Filter the triage state of a finding. Refer to [Triage statuses](/semgrep-code/triage-remediation#triage-statuses) to understand triage states. |
 | **Severity**           | Filter by the severity of a finding. Severity is computed based on the values assigned for [Likelihood](/contributing/contributing-to-semgrep-rules-repository/#likelihood) and [Impact](/contributing/contributing-to-semgrep-rules-repository/#impact) by the rule's author. Possible values: <ul><li>Low</li><li>Medium</li><li>High</li></ul> |
 | **Confidence**         | Filter by the likelihood of the rule to detect true positives. The higher the confidence, the more true positives the rule may detect. |
-| **Pro findings**       | Filter for findings identified using Semgrep Pro rules. |
+| **Pro findings**       | Filter for findings identified using Semgrep Pro rules. Also includes findings originating from Pro Engine analysis; if Semgrep identifies a finding as the result of interfile analysis, it's included by this filter. |
```

Comment:
this may not necessarily be interfile analysis, it could be interprocedural. this is kind of a frustrating thing to phrase but it's basically "things which could only be found as a result of Pro Engine analysis"